openedx/features/badging/helpers.py

```diff
@@ -116,3 +120,31 @@ def filter_earned_badge_by_joined_team(user, course, earned_badges):
         earned_badge for earned_badge in earned_badges if
         earned_badge.community_id == course_team[TEAM_ROOM_ID_KEY]
     ]
+
+
+def send_user_badge_notification(user, my_badge_url, badge_name):
+    """
+    Send user new badge notification
+    :param user: Notification sender user
+    :param my_badge_url: Redirect url on notification click
```

Comment:
Let's change this to `:param my_badge_url: Redirect url to my_badge view on notification click`

Reply:
done